core/src/ast/mod.rs

```diff
@@ -42,7 +42,59 @@ pub struct ForeignKey {
 #[derive(PartialEq, Debug, Clone, Eq, Hash, Serialize, Deserialize, Display)]
 pub enum ReferentialAction {
     #[strum(to_string = "NO ACTION")]
+    /// A NO ACTION constraint specifies that when a referenced row is deleted,
+    /// no action should be taken.
     NoAction,
+    #[strum(to_string = "CASCADE")]
+    /// A CASCADE constraint specifies that when a referenced row is deleted,
+    /// row(s) that reference it should also be deleted.
+    Cascade,
+    #[strum(to_string = "SET NULL")]
+    /// A SET NULL constraint specifies that when a referenced row is deleted,
+    /// row(s) that reference it should have their referencing column(s) set to NULL.
+    SetNull,
+    #[strum(to_string = "SET DEFAULT")]
+    /// A SET DEFAULT constraint specifies that when a referenced row is deleted,
+    /// row(s) that reference it should have their referencing column(s) set to the column's default value.
+    SetDefault,
+    #[strum(to_string = "RESTRICT")]
+    /// A RESTRICT constraint specifies that when a referenced row is attempted to be deleted,
+    /// the deletion should be restricted i.e. prevented.
+    Restrict,
```

Comment:
We decided merge `NO ACTION and RESTRICT` to NO ACTION so that it would be better to remove Restrict variant here.

Reply:
Ok!

Reply:
Done!